src/Blocks/BlockOre.h

```diff
@@ -35,17 +95,19 @@ class cBlockOreHandler:
 			}
 		}
 
-		// TODO: Handle the Fortune enchantment here
+		unsigned int fortuneLevel = ToolFortuneLevel(a_Tool);
+		unsigned int dropMult = fortuneDropMult(fortuneLevel);
```

Comment:
These two should be `const`, and they can also be `auto`. It's also our style to use UpperCamelCase for local variable names (although I personally prefer camelCase).